addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -53,6 +57,11 @@ export class PivotCorePlugin extends CorePlugin {
         };
     }
 
+    /**
+     * @param {AllCoreCommand} cmd
+     *
+     * @returns {string | string[]}
+     */
```

Comment:
nitpick but i'd rather we have a clear guideline (that we probably won't follow): do we want to have empty lines between `@params` and `@returns` in our jsDoc ? I perosnnaly don't like it.

Reply:
Personnally I don't care :D , but my editor automatically add a space so...
Anyway, I think that you're right, we probably won't follow any guideline we define for doc